include/zephyr/bluetooth/mesh/access.h

```diff
@@ -824,9 +831,18 @@ struct bt_mesh_model *bt_mesh_model_find_vnd(const struct bt_mesh_elem *elem,
  */
 static inline bool bt_mesh_model_in_primary(const struct bt_mesh_model *mod)
 {
-	return (mod->elem_idx == 0);
+	return (mod->ctx->elem_idx == 0);
 }
 
+/** @brief Get unicast address of the element.
```

Comment:
```suggestion
/** @brief Get the address of the element.
```
Elements have only unicast addresses.